vectorsearch/test_procedures/common/train-model-schedule.json

```diff
@@ -29,7 +37,7 @@
 
                         {%- if faiss_nprobes is defined and faiss_nprobes %}
                             {%- if faiss_nlist is defined and faiss_nlist %}
```

Comment:
Please document the `faiss_nprobes`, `faiss_nlist`, etc. Jinja variables in the README, so that folks using the `faiss`engine are aware of them.